neuralmonkey/learning_utils.py

```diff
@@ -223,9 +223,10 @@ def training_loop(sess, saver,
 
                 if step % validation_period == validation_period - 1:
                     decoded_val_sentences, decoded_raw_val_sentences, \
-                        val_evaluation = run_on_dataset(
+                        val_evaluation, val_images = run_on_dataset(
```

Comment:
Considering that the inputs of NM can be also images (we did image captioning with it previously), name `images` may be confusing. What about `plots`, `visualizations`, ... I don't know, maybe even images are OK.